Assignment4/Assignment4.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs');
+
+function readAppendWrite (readFile, appendFile, writeFile) {
+  fs.readFile (readFile, function (err1, data1) {
```

Comment:
`err1` and `data1`  can be a bit descritive, like `errorOnFile1Read`, `dataFromFile1` and similarly for other error and data parameter.
Problem Type: Naming Convention